python.spec

```diff
@@ -1,4 +1,4 @@
-### RPM external python 2.7.6
+### RPM external python 2.7.10
```

Comment:
Moving to the new python should be checked with computing. You should also pick up sources from the `cms-externals/cpython` repository, not the official one.